CHANGES.md

```diff
@@ -1,5 +1,371 @@
 # Changelog
 
+## 1.10.7 (core) / 0.26.7 (libraries)
+
+### New
+
+- Applying changes from sensor test results now also applies changes from dynamic partition requests.
+- When merging assets from multiple code locations, autogenerated specs are now prioritized lower than customized external asset specs.
+- [ui] Allowed using command-click to view a run from the toast message that appears when starting a materialization of an asset.
+- [ui] Asset graph can now zoom out a lot more.
+- [ui] Added a kind tag for dbt Cloud.
+- [dagster-dlt] Added backfill policy to dlt_assets, defaulting to single-run. (Thanks [@neuromantik33](https://github.com/neuromantik33)!)
+
+### Bugfixes
+
+- Updated `AutomationCondition.initial_evaluation` condition to become true for all partitions of an asset whenever the PartitionsDefinition of that asset changes, rather than whenever the structure of the condition changes.
+- Fixed a bug with several integrations that caused data fetched from external APIs not to be properly cached during code server initialization, leading to unnecessary API calls in run and step worker processes. This affected `dagster-airbyte`, `dagster-dlift`, `dagster-dbt`, `dagster-fivetran`, `dagster-looker`, `dagster-powerbi`, `dagster-sigma`, and `dagster-tableau`.
+- [ui] Fixed an issue with the Run step selection input autocomplete where it would suggest `key:"*substring*"` instead of `name:"*substring*"`.
+- [ui] Fixed the "View run" link shown when launching runs.
+- [ui] Fixed an issue where updating a catalog view caused an infinite loading state.
+- Fixed an issue which could cause asset check evaluations emitted from the body of the op to not impact the check status of an asset in the UI.
+- Fixed an issue that could cause an asset backfill created by re-executing another backfill from the point of failure to error on the first tick in rare cases.
+- Fixed an issue that could cause automation condition evaluations to fail to render in the UI in rare cases.
+- [ui] Fixed a regression in the "Cancel Backfill" option for job backfills that have finished queuing runs.
+- [ui] Fixed overflow of long runs feed table on backfill page.
+- [dagster-dbt] Replaced `@validator` with `@field_validator` in dagster_dbt/core/resource.py to prevent Pydantic deprecation warnings. (Thanks [@tintamarre](https://github.com/tintamarre)!)
+
+### Documentation
+
+- Updated the "Asset versioning and caching" guide to reflect the current Dagster UI and "Unsynced" labels.
+- Removed a few repeated lines in documentation on customizing automation conditions. (Thanks [@zero-stroke](https://github.com/zero-stroke)!)
+- Fixed example in TableRecord documentation to use the new input format.
+
+### Configuration
+
+- [dagster-gcp] Updated Dataproc configuration to the latest version. If necessary, consider pinning your `dagster-gcp` version while you migrate config. Please see the full list of changed fields below.
+
+  **Dataproc job config**
```

Comment:
Thoughts on separating this block and the following block into a github discussion or something?